experimental/traffic-portal/src/app/core/users/tenants/tenants.component.ts

```diff
@@ -122,6 +114,15 @@ export class TenantsComponent implements OnInit, OnDestroy {
 						multiRow: true,
 						name: "Disable"
 					});
+					this.contextMenuItems.push({
+						href: (t: Tenant): string => `core/tenants/${t.id}`,
+						name: "View Details"
+					});
+					this.contextMenuItems.push({
+						href: (t: Tenant): string => `core/tenants/${t.id}`,
+						name: "Open in New Tab",
+						newTab: true
+					});
```

Comment:
I don't know why, but I can't get a context menu to appear on the Tenant's table anymore. This is the only thing that would've changed in that regard, so I feel like it has to be the problem, but I can't really see why that would be, to be honest.